source/blog/2016-07-25-ember-2-7-and-2-8-beta-released.md

```diff
@@ -0,0 +1,194 @@
+    },{
+      description: 'Aliens',
+      car: { id: 2, make: 'Toyota' }
+    },{
+      description: 'Road rage',
+      car: { id: 1, make: 'Saturn' }
+    }];
+  }),
+  carsInAccidents: Ember.computed.mapBy('accidents', 'car'),
+  uniqueCarsInAccidents: Ember.computed.uniqBy('carsInAccidents', 'id')
+});
+```
+
+`uniqBy` is also added as an array method. For example:
+
+```js
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  accidents: Ember.computed(function() {
+    return [{
+      description: 'Flipped over',
+      car: { id: 1, make: 'Saturn' }
+    },{
+      description: 'Aliens',
+      car: { id: 2, make: 'Toyota' }
+    },{
+      description: 'Road rage',
+      car: { id: 1, make: 'Saturn' }
+    }];
+  }),
+  uniqueCarsInAccidents: Ember.computed('accidents.@each.car', function() {
+    return this.get('accidents').mapBy('car').uniqBy('id');
+  })
+});
+```
+
+For more information on these methods, see the [API docs for the computed
+macro](http://emberjs.com/api/classes/Ember.computed.html#method_uniqBy), [API
+docs for the array
+method](http://emberjs.com/api/classes/Ember.Array.html#method_uniqBy), and [PR
+#12875](https://github.com/emberjs/ember.js/pull/12875) introducing these
+changes.
+
+#### Notable Deprecations
+
+The following deprecations are scheduled for release with Ember 2.7 and will be
+removed soon:
+
+* `Ember.Backburner` was private throughout the Ember 2.x series and will be
+  removed after Ember 2.8, as mentioned in the
+  [deprecation guide](http://emberjs.com/deprecations/v2.x/#toc_ember-backburner).
+
+* `Ember.Binding` has not been needed for some time and is deprecated in favor
+  of computed properties and services (depending on what you were binding to).
+  It will be removed in Ember 3.0. See the
+  [deprecation guide](http://emberjs.com/deprecations/v2.x/#toc_ember-binding)
+  for more details.
+
+For more details on changes landing in 2.7, review the
+[Ember.js 2.7.0 CHANGELOG](https://github.com/emberjs/ember.js/blob/v2.7.0/CHANGELOG.md).
+
+### Ember.js 2.8 Beta
+
+Ember 2.8 beta introduces the following new backward compatible APIs:
+
+#### Engines
+
+As described in the [RFC](https://github.com/emberjs/rfcs/pull/10), "engines
+allow multiple logical applications to be composed together into a single
+application from the user's perspective."
+
+Work on engines has proceeded over the past year in both Ember core as well as
+the [ember-engines](https://github.com/dgeb/ember-engines) addon. Ember's
+architecture has been enhanced to provide the base classes and hooks needed to
+support engines. The addon then makes use of these interfaces to provide a
+smooth experience building and consuming engines.
+
+Until now, none of the engine-related classes and hooks in Ember have been
+exposed publicly. However, starting with Ember 2.8, the following APIs will be
+made public:
+
+* `Engine` class - The base class for `Application`. Its only public
```

Comment:
Should this be `Ember.Engine`, to make it clearer where it is exposed?  Or should we hide the globals API?